server/internal/mcp/rpc_tools_call.go

```diff
@@ -111,11 +138,6 @@ func handleToolsCall(
 		ciEnv.Set(k, v)
```

Comment:
What is supposed to happen when a user provides an env var that overrides one from an attached env?

Reply:
Perhaps maliciously